LoRaEngine/LoraKeysManagerFacade/IDeviceTwin.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using System;
+
+    public interface IDeviceTwin
+    {
+        string DeviceId { get; }
+
+        string GetGatewayID();
+
+        string GetNwkSKey();
+
+        string GetDevAddr();
+
+        DateTime GetLastUpdated();
```

Comment:
Should this be in local time always? Why not just go with `DateTimeOffset` to have the least ambiguity?

Reply:
Before this PR, it used Local DateTime ...